src/pages/LogOutPreviousUserPage.js

```diff
@@ -36,18 +36,16 @@ class LogOutPreviousUserPage extends Component {
             // and their authToken stored in Onyx becomes invalid.
             // This workflow is triggered while setting up VBBA. User is redirected from NewDot to OldDot to set up 2FA, and then redirected back to NewDot
             // On Enabling 2FA, authToken stored in Onyx becomes expired and hence we need to fetch new authToken
-            const shouldForceLogin = lodashGet(this.props, 'route.params.shouldForceLogin', '') === 'true';
+            const shouldForceLogin = lodashGet(props, 'route.params.shouldForceLogin', '') === 'true';
             if (shouldForceLogin) {
-                const email = lodashGet(this.props, 'route.params.email', '');
-                const shortLivedAuthToken = lodashGet(this.props, 'route.params.shortLivedAuthToken', '');
+                const email = lodashGet(props, 'route.params.email', '');
+                const shortLivedAuthToken = lodashGet(props, 'route.params.shortLivedAuthToken', '');
                 Session.signInWithShortLivedAuthToken(email, shortLivedAuthToken);
             }
         });
-    }
+    }, [props]);
```

Comment:
> Try to avoid adding props as a dependency of useEffect or useCallback since it will trigger whenever any prop changes even though the code may be referencing only 1 or 2 props inside the hook. [The ](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#what-is-the-exhaustive-deps-lint-rule-can-i-ignore-it)exhaustive-deps[ lint rule](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#what-is-the-exhaustive-deps-lint-rule-can-i-ignore-it) can be of help here.

Source: https://expensify.slack.com/archives/C01GTK53T8Q/p1686062674061049?thread_ts=1685727880.152119&cid=C01GTK53T8Q